.github/workflows/build.yml

```diff
@@ -37,14 +37,11 @@ jobs:
         key: ${{ runner.os }}-${{ matrix.node-version }}-node_modules-${{ hashFiles('**/package-lock.json') }}
     - run: npm install
     - run: npm run check
-    - name: Install Python packaging & linting tools
-      run: python -m pip install -r dev-requirements.txt -r requirements.txt
-    - run: mypy sphinx_immaterial/**/*.py tests/**/*.py
-      shell: bash
-    - run: pylint sphinx_immaterial/**/*.py
-      shell: bash
-    - run: black --diff --color sphinx_immaterial/**/*.py tests/**/*.py
-      shell: bash
```

Comment:
FYI, using bash on Windows runners ends up using WSL which is slower than PowerShell. I try to avoid overusing this.